src/components/pages/auth/profile/index.js

```diff
@@ -0,0 +1,177 @@
+        passwordPlaceholder: {
+            id: 'form.password.placeholder',
+            defaultValue: 'Enter your password',
+        },
+    });
+
+    const getContainerClassNames = () => classnames('rainbow-auth-firebase_profile', className);
+
+    const handleProfileChange = (profile) => {
+        updateProfile(profile);
+    };
+
+    return (
+        <section className={getContainerClassNames()} style={style}>
+            <div className="rainbow-auth-firebase-profile_top-bar">
+                <span className="rainbow-auth-firebase-profile_top-bar-title">
+                    <FormattedMessage id="profile.title" defaultMessage="Edit Profile" />
+                </span>
+                <Link to="/app/home">
+                    <ButtonIcon icon={<CloseIcon className="rainbow-auth-firebase-profile_top-bar-close-icon" />} />
+                </Link>
+            </div>
+            <div className="rainbow-auth-firebase-profile_content">
+                <div>
+                    <Field
+                        component={Input}
+                        name="displayName"
+                        className="rainbow-auth-firebase-profile_content-input"
+                        label={<FormattedMessage id="form.displayName.label" defaultMessage="Name" />}
+                        required
+                        placeholder={intl.formatMessage(translations.displayNamePlaceholder)}
+                        icon={<UserIcon />} />
+                    <Field
+                        component={Input}
+                        name="email"
+                        className="rainbow-auth-firebase-profile_content-input"
+                        label={<FormattedMessage id="form.email.label" defaultMessage="Email address" />}
+                        required
+                        placeholder={intl.formatMessage(translations.emailPlaceholder)}
+                        icon={<EmailIcon />} />
+                    <Field
+                        component={Input}
+                        name="phoneNumber"
+                        className="rainbow-auth-firebase-profile_content-input"
+                        label={<FormattedMessage id="form.phone.label" defaultMessage="Phone number" />}
+                        placeholder={intl.formatMessage(translations.phonePlaceholder)}
+                        icon={<PhoneIcon />} />
+                    <Field
+                        component={Input}
+                        name="password"
+                        label={<FormattedMessage id="form.password.label" defaultMessage="Password" />}
+                        placeholder={intl.formatMessage(translations.passwordPlaceholder)}
+                        type="password"
+                        className="rainbow-auth-firebase-profile_content-input"
+                        icon={<LockIcon />} />
+                </div>
+                <div className="rainbow-auth-firebase-profile_content-user-photo-container">
+                    <span>
```

Comment:
if this span is not needed remove it because FormattedMessage already have a span